ludwig-conf-sample.js

```diff
@@ -1,12 +1,13 @@
 module.exports = {
 	repo: 'github-user/repository',
-	acceptedTestsLocation:'/',
-	github:{
-		branch:'master',
-		authenticationCallback:'http://authentication.callback.url/github_callback'
+	acceptedTestsLocation: '/',
+	accessControlAllowOrigin: 'http://localhost:3000',
+	github: {
+		branch: 'master',
+		authenticationCallback: 'http://authentication.callback.url/github_callback'
```

Comment:
Any chance that the host for `accessControlAllowOrigin` and `authenticationCallback` differ?

It seems to me there should be a `host` property that is used by default for both values.